### ISSUE TITLE
Cargo.toml references sdl2 fork branch 'sensor-api'

### ISSUE THREAD

**Issue by theryangeary** [Yamakaky/gyromouse#30]
but 'sensor-api' does not exist on the fork: https://github.com/Yamakaky/rust-sdl2/branches/all

**Comment by Yamakaky** [comment]
Yes, it was an old workaround that was merged upstream. I started to remove this plus a few other workarounds, I need to push it.

**Comment by Yamakaky** [comment]
Hey, sorry for the long delay. I pushed a version with updated dependencies, it should fix your issue.
